Makefile

```diff
@@ -219,7 +219,7 @@ uninstall: manifests ## Uninstall CRDs from the K8s cluster specified in ~/.kube
 
 .PHONY: deploy
 deploy: manifests ## Deploy controller to the K8s cluster specified in ~/.kube/config.
-	cd deploy/controller && kubectl apply -f crds.yaml -f rbac.yaml -f setup-controller.yaml
+	cd deploy/controller && kubectl apply -f setup-controller.yaml -f crds.yaml -f rbac.yaml
```

Comment:
can you please update why this change is required?

Reply:
TLDR; the namespace creation moved from `config/rbac/` to `config/manager/`

The namespace is created in config/manager now. As everything else is namespaces, the namespace needs to exist first.

Not following this order will error out on first try, but will succeed in subsequent tries (coz the ns is created later in the process by setup-controller.yaml).